teamserver/teamserver/events/worker.py

```diff
@@ -42,7 +45,9 @@ def notify_integration(integration, data):
     Trigger an integration with the given data.
     """
     if integration and isinstance(integration, Integration):
-        integration.run(data)
+        # Check if the integration is enabled
+        if integration.config.get("enabled", False):
```

Comment:
Check to ensure that 'config' exists, to ensure not all integrations break if one is configured incorrectly.

Reply:
You could add an abstract field to the integration class to achieve this